pkg/storaged/overview.js

```diff
@@ -476,6 +478,23 @@
                 return this_deferred.promise();
             }
 
+            var mount_options = entry? entry.fields[3] : "defaults";
+            var split_options = format.parse_options(mount_options == "defaults" ? "" : mount_options);
+            var opt_auto = !format.extract_option(split_options, "noauto");
+            var opt_ro = format.extract_option(split_options, "ro");
```

Comment:
To  be sure: `defaults` is just a "visible" name for an empty set of mount options -- i. e. if the mount options are `ro`, then the implied default is still `auto`, right?

Reply:
Yes.